### PR TITLE
feat: [CDS-39415]: added scope info to template card

### PR DESCRIPTION
**Pull request by yc147**
#### Screenshots
Screenshot 2022-07-05 at 4 18 04 PM
Screenshot 2022-07-05 at 4 18 20 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/72-templates-library/pages/TemplatesPage/views/TemplatesListView/TemplatesListView.tsx [hunk 1]
```diff
@@ -79,36 +81,43 @@ const RenderColumnTemplate: Renderer<CellProps<TemplateSummaryResponse>> = ({ ro
...
+                  <br />
+                  <Text lineClamp={1}>
+                    {getString('descriptionLabel', { description: defaultTo(data.description, '-') })}
+                  </Text>
+                </Layout.Vertical>
+              }
+            >
+              {data.name}
+            </Text>
+            {data.tags && !isEmpty(data.tags) && <TagsPopover tags={data.tags} />}
```
Comment (samarth-sharma31) [inline src/modules/72-templates-library/pages/TemplatesPage/views/TemplatesListView/TemplatesListView.tsx:109]: data.tags --> check is redundant --> only !isEmpty(data.tags) should be fine i guess

File: src/modules/72-templates-library/components/TemplateCard/TemplateCard.tsx [hunk 2]
```diff
@@ -99,6 +100,9 @@ export function TemplateCard(props: TemplateCardProps): JSX.Element {
             {getString('idLabel', { id: template.identifier })}
           </Text>
         </Container>
+        <Container>
```
[the remark this answers is not in the document]
Comment (yc147) [reply src/modules/72-templates-library/components/TemplateCard/TemplateCard.tsx:103]: Yes

File: src/modules/72-templates-library/pages/TemplatesPage/views/TemplatesListView/TemplatesListView.tsx (hunk 1 above)
Comment (yc147) [reply src/modules/72-templates-library/pages/TemplatesPage/views/TemplatesListView/TemplatesListView.tsx:109]: This code is not new.

File: src/modules/10-common/components/ScopeBadge/ScopeBadge.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+import React from 'react'
+import cx from 'classnames'
+import { Tag } from '@wings-software/uicore'
+import type { ScopedObjectDTO } from '@common/components/EntityReference/EntityReference'
+import { getScopeFromDTO } from '@common/components/EntityReference/EntityReference'
+import css from './ScopeBadge.module.scss'
+
+interface ScopeBadeProps {
```
[the remark this answers is not in the document]
Comment (yc147) [reply src/modules/10-common/components/ScopeBadge/ScopeBadge.tsx:15]: fixing it.

**Review by samarth-sharma31 — DISMISSED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**